src/documents/InclusiveLearning.html.md.handlebars

```diff
@@ -39,7 +39,7 @@ Learning needs that affect learning can include:
 <figure>
 [![Floe concept design mindmap]({{{getRelativeUrl '/images/thumbs/400px-Floe_concept_design_mindmap.png' document.url}}})]({{{getRelativeUrl '/images/Floe_concept_design_mindmap.png' document.url}}})
 <figcaption>
-*Figure:* Floe concept design mindmap
+*Figure:* Floe concept design mindmap illustrates the many different ideas and parts which work together to enable a one-size-fits-one education.
```

Comment:
Do we need "*Figure:*"? It's semantically attached with `figure` and `figcaption` elements, and also is visually styled to be related to the image.